testlib/src/main/java/com/diffplug/spotless/NonSerializableList.java

```diff
@@ -1,46 +0,0 @@
-/*
- * Copyright 2016 DiffPlug
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.diffplug.spotless;
-
-import java.util.AbstractList;
-import java.util.Arrays;
-import java.util.Objects;
-
-public final class NonSerializableList extends AbstractList<String> {
```

Comment:
This class wasn't used anywhere except in a JUnit 3 test. So I removed both the class and the test. Hope that's okay :)

Reply:
Lol, good call. Kill the cruft!